treeherder/perf/models.py

```diff
@@ -580,6 +580,15 @@ class BackfillRecord(models.Model):
     def id(self):
         return self.alert
 
+    def get_pushes_in_context_range(self):
```

Comment:
We should also use this method in `perf.email.BackfillReportContent.__fetch_border_pushes()`.